integration-test/src/main.rs

```diff
@@ -0,0 +1,243 @@
+use mongodb::{
```

Comment:
Regarding the directory structure, we may want another directory for the dataloader I'm thinking because we'll also have one more executable for the test generator.  Not sure yet how Rust would run them separately, but something like `integration-test/src/dataloader`, `integration-test/src/test-runner`, `integration-test/src/test-generator` is one suggestion.

Reply:
That's a good breakdown, I'll set it up that way 👍